scripts/create-update-issues.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DEFAULT_REF="HEAD"
+DEFAULT_LABEL="client-controller-update"
+
+print-usage() {
+  cat <<EOT
+Detects major-bumped packages in a release commit and creates tickets in clients
+that instructs engineers to upgrade to them.
+
+Usage: $0 [--ref REF] [--no-dry-run]
+
+OPTIONS:
+
+--ref REF, -r REF       The release commit to inspect.
+--no-dry-run            By default, this script won't do anything. Pass this
+                        option to override that.
+--help, -h              You're looking at it ;)
+EOT
+}
+
+existing-issue-found() {
+  local repo="$1"
+  local package_name="$2"
+  local version="$3"
+
+  local all_issues
+  if ! all_issues=$(gh issue list --repo "$repo" --label "$DEFAULT_LABEL" --state open --json number,title,url 2>&1); then
+    echo "❌ Failed to fetch issues from ${repo}"
+    echo "$all_issues"
+    exit 1
+  fi
+
+  if [[ -z "$all_issues" || "$all_issues" == "[]" ]]; then
+    echo "Found no issues in the repo."
+    return 1
+  fi
+
+  local package_name_without_at="${package_name#@}"
+  local package_name_escaped="${package_name//./\\.}"
+  local package_name_without_at_escaped="${package_name_without_at//./\\.}"
+  local regex_pattern="^(Update|Upgrade) (${package_name_escaped}|${package_name_without_at_escaped}) to version ${version//./\\.}$"
```

Comment:
Curious, did we change the issue names at some point? I see we're matching with and without the `@`, and with either Update or Upgrade

Reply:
Yes, this PR improves the issue names as well. I've always found the existing terminology to be slightly annoying. I feel that "upgrade" is a better word to use than "update" (which could imply changing the package, adding a patch, or something like that). Also, `@` was somehow missing from the issue title. So you'd get something like "Update metamask/foo to version 1.2.3" which seemed strange. Now it says "Upgrade @metamask/foo to 1.2.3". Maybe minor, but I figure it could help.